modules/components/babel.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
renaming this file is something required when linking the package inside nextjs using `next-transpile-modules`